container_files/public_html/doc/builtin/procedures/Procedures.md

```diff
@@ -35,6 +35,48 @@ A procedure may return results as follows:
   may contain a more detailed set of information about what was done including elements like
   logs of messages and errors.
 
+## Getting progress of a procedure
```

Comment:
Et il manque un truc: This feature is not currently implemented for all procedures. 

et pe lister où c'est fait?